src/docs/faq.md

```diff
@@ -1,70 +1,99 @@
 
 #### Can I use one driver license across multiple devices?
 * No. One license per device.
 
-#### I want to buy a Chromebook to run Windows or Linux or macOS -- which should I buy?
-* Buying any Chromebook with the intention of running macOS, Windows or Linux is not a great idea. -- DO NOT EXPECT EVERYTHING TO WORK OUT OF THE BOX. Older models may fair better compatibility wise, but there are still lots of caveats, and it's not recommended to buy a Chromebook as a cheap Linux device.
+<br>
 
-#### How can I get audio working under Linux?
-* Please see [here](https://github.com/WeirdTreeThing/chromebook-linux-audio)
+## Linux Questions
 
-#### How do I get my top row keys on Linux to act like how they did on chromeOS?
-* Follow the instructions [here](https://github.com/WeirdTreeThing/cros-keyboard-map).
+### How come audio isn't working on Ubuntu / Ubuntu forks?
+* Ubuntu and Ubuntu-based distros may have issues and are not supported, try switching to another distro. Debian is a great alternative if you are used to ubuntu.
 
-#### I just want to boot Linux from USB on my Chromebook, what do I need to do?
-* Check the Supported Devices page, to ensure your device has functional RW_LEGACY firmware available. If so, flash RWL (RW_LEGACY firmware) and use that.
+### I want to buy a Chromebook to run Windows or Linux or macOS -- which should I buy?
+* If you're looking to buy for macOS, you will need a chromebook with an 10th generation intel cpu or lower, Celeron/Pentium are not supported. Any 64 bit chromebook will work in Windows and Linux. Some require paid drivers (Windows only).
 
-#### I just installed the UEFI firmware, and now my device boots to a black screen that says 'shell' - what do I do?
-* You're in the EFI shell; the firmware boots there when it can't find a valid UEFI boot device, either externally (USB, SD) or internally. Just type `exit` to get back to the the UEFI settings menu.
+### What Linux distros are recommended?
+* See [this page](installing/installing-linux.md).
 
-#### Should I install 32 or 64-bit OS?
-* The UEFI Full ROM firmware only supports 64-bit OS, so be sure to use the 'x86_64' version of whatever OS you want to install. USB media created from 32-bit ISOs will simply fail to boot.
+### How can I get audio working under Linux?
+* Please see [this github repo](https://github.com/WeirdTreeThing/chromebook-linux-audio)
 
-#### I'm using your UEFI firmware, installed my OS [Linux], and it still boots to the EFI shell - what do I do?
+### How do I get my top row keys on Linux to act like how they did on chromeOS?
+* Follow the instructions [in this github repo](https://github.com/WeirdTreeThing/cros-keyboard-map).
+
+### I'm using your UEFI firmware, installed my OS [Linux], and it still boots to the EFI shell - what do I do?
```

Comment:
replace your with the

Reply:
1. Not Chromebook/box specific
2. Not all DIstros use Grub

I'd say be gone with it

Reply:
Great idea. Its not chromebook specific related anyways

Reply:
> 1. Not Chromebook/box specific

I believe it was copied over from mrchromebox.tech/#faq